src/rust/src/http_reader.rs

```diff
@@ -85,7 +91,7 @@ impl HttpFgbReader {
         self.fbs.header()
     }
     fn header_len(&self) -> usize {
-        12 + self.fbs.header_buf.len()
+        8 + self.fbs.header_buf.len()
```

Comment:
I think there is logic elsewhere that assumes this length includes the length i32 is included, so I think you'll also need to update some users, like `select_bbox`.

Reply:
Not sure I understand. `header_len` *does* still include the length i32, but the length i32 is now included in `fbs.header_buf`

Reply:
Oh, my mistake! Thanks for clarifying.